Changelog.md

```diff
@@ -12,7 +12,7 @@ Compiler Features:
 
 Bugfixes:
  * Yul / Inline Assembly Parser: Disallow trailing commas in function call arguments.
- * Set state mutability of the function type members ``gas`` and ``value`` to pure (while their return type inherits state mutability from the function type).
+ * Type Checker: Set state mutability of the function type members ``gas`` and ``value`` to pure (while their return type inherits state mutability from the function type).
```

Comment:
The bugfixes and build system paragraphs (not in PR) are not alphabetically sorted.

Reply:
Should be fixed now.